server/views/DB.py

```diff
@@ -0,0 +1,7 @@
+from pymongo import MongoClient
+
+def getBaseColl():
+    mongoClient = MongoClient("mongodb://localhost:27017")
```

Comment:
Tip: this connection string could be retrieved from a file or environment variable.